{"name": "test_contact_us", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <tests.TestHRM object at 0x045F3628>\n\n    def test_contact_us(self):\n        x = datetime.now()\n        self.driver.get(\"https://smoothapps.com/\")\n        textContent = self.driver.find_element_by_xpath(\"//*[@id='et-secondary-nav']/li[1]/a\")\n        print('time test_contact_us {}'.format(datetime.now() - x))\n        if textContent.text=='ERROR':\n            assert True\n        else:\n>           assert False\nE           assert False\n\ntests.py:70: AssertionError"}, "start": 1586883238209, "stop": 1586883242438, "uuid": "d7b169b4-edf6-435d-bfca-daa8a8bbbbaa", "historyId": "bd3e9f95747a153ec240119dad03be0d", "testCaseId": "39e9083f1e845958efa2deaac5e3ca68", "fullName": "tests.TestHRM#test_contact_us", "labels": [{"name": "severity", "value": "minor"}, {"name": "suite", "value": "tests"}, {"name": "subSuite", "value": "TestHRM"}, {"name": "host", "value": "DESKTOP-5N2TE77"}, {"name": "thread", "value": "22308-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests"}]}